packages/mobile/src/screens/upload-screen/screens/CompleteTrackScreen.tsx

```diff
@@ -36,12 +36,15 @@ export const CompleteTrackScreen = () => {
     [navigation, track, uploadAttempt]
   )
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const metadata = useMemo(() => track?.metadata, [])
```

Comment:
why is this necessary?
feels like disabling reinitialize should cover it

Reply:
you might be right, I'll check

Reply:
i actually just went over this again and realized I don't need any of this stuff, fixing now